Ensure that eselect uses a canonical path for Bash
which ensures eselect has the correct "shebang" when
installed during Funtoo From Scratch (ffs) stage1 builds
https://bugs.funtoo.org/browse/FL-10625
--- a/configure.ac
+++ b/configure.ac
@@ -12,7 +12,7 @@
 # the /bin/sh symlink to a non-bash shell, all hell will break lose.
 # thanks to James Rowe for the heads up.
 unset BASH
-AC_PATH_PROGS(BASH, bash)
+AC_SUBST(BASH, /bin/bash)
 if test x$BASH = x; then
     AC_MSG_ERROR([bash is required])
 fi
